teku/src/main/java/tech/pegasys/teku/cli/subcommand/debug/DebugDbCommand.java

```diff
@@ -355,6 +357,66 @@ public int dumpHotBlocks(
     return 0;
   }
 
+  @Command(
+      name = "dump-blobs-sidecar",
+      description = "Writes all blobs sidecars in the database as a zip of SSZ files",
+      mixinStandardHelpOptions = true,
+      showDefaultValues = true,
+      abbreviateSynopsis = true,
+      versionProvider = PicoCliVersionProvider.class,
+      synopsisHeading = "%n",
+      descriptionHeading = "%nDescription:%n%n",
+      optionListHeading = "%nOptions:%n",
+      footerHeading = "%n",
+      footer = "Teku is licensed under the Apache License 2.0")
+  public int dumpBlobsSidecar(
+      @Mixin final BeaconNodeDataOptions beaconNodeDataOptions,
+      @Mixin final Eth2NetworkOptions eth2NetworkOptions,
+      @Option(
+              required = true,
+              names = {"--output", "-o"},
+              description = "File to write blobs sidecars to")
+          final Path outputFile)
+      throws Exception {
+    int index = 0;
+    try (final Database database = createDatabase(beaconNodeDataOptions, eth2NetworkOptions);
+        final ZipOutputStream out =
+            new ZipOutputStream(
+                Files.newOutputStream(
+                    outputFile, StandardOpenOption.CREATE, StandardOpenOption.TRUNCATE_EXISTING))) {
+
+      try (final Stream<Map.Entry<SlotAndBlockRoot, Bytes>> streamBlobsSidecarAsSsz =
+          database.streamBlobsSidecarsAsSsz(UInt64.ZERO, UInt64.MAX_VALUE)) {
```

Comment:
Wouldn't it be helpful to make these params overridable?

Reply:
I ended up implementing it